dbt/models/ferc714/out_ferc714__hourly_planning_area_demand/schema.yml

```diff
@@ -7,13 +7,43 @@ sources:
           - check_row_counts_per_partition:
               table_name: out_ferc714__hourly_planning_area_demand
               partition_column: report_date
+          - expect_unimputed_values_are_unchanged:
+              original: demand_reported_mwh
+              imputed: demand_imputed_pudl_mwh
+              imputation_code: demand_imputed_pudl_mwh_imputation_code
+              atol: 1.0
+              rtol: 0.8
```

Comment:
Unfortunately I was only able to get this down to 0.8. The problem is that there are some reported values with very low magnitude (< 1.0 MWH) which also have a mismatch between imputed/reported, so even though the absolute error is not too high, the relative error is.